models/stg_linkedin.yml

```diff
@@ -17,6 +18,36 @@ models:
 
       - name: date_day
         description: The day of the activity.
+        tests:
+          - not_null
+
+      - name: clicks
+        description: The count of chargeable clicks.
+
+      - name: impressions
+        description: This is the count of "impressions" for Direct Ads and Sponsored Updates and "sends" for InMails.
+
+      - name: cost
+        description: The cost of the ads in the local currency or USD.
+
+  - name: stg_linkedin_ads__ad_analytics_by_campaign
+    description: Each record in this table represents the daily performance of ads at the campaign level.
+    columns:
+      - name: daily_campaign_id
+        description: A unique key, a combination of creative_id and date_day.
+        tests:
+          - unique
+          - not_null
```

Comment:
For all other packages in these updates we opted to not use the surrogate key and instead leverage the unique combination of columns test. Let's bring this up during standup to discuss as a team.

Reply:
I see this was used across the staging models. Whatever we discuss during standup can be either applied or kept for all instances.

Reply:
removing and using dbt_utils.unique_combination_of_columns